presto-docs/src/main/sphinx/connector/localfile.rst

```diff
@@ -0,0 +1,45 @@
+====================
+Local File Connector
+====================
+
+The local file connector allows querying data stored on the local
+filesystem of each worker.
+
+Configuration
+-------------
+
+To configure the local file connector, create a catalog properties file
+``etc/catalog/localfile.properties`` with the following contents:
+
+.. code-block:: none
+
+    connector.name=localfile
+
+Using the Local File Connector
+------------------------------
+List the available system schemas::
```

Comment:
I'm not sure this section is useful at all. Does the connector even have multiple schemas?